README.md

```diff
@@ -6,6 +6,8 @@
 
 ExDoc is a tool to generate documentation for your Elixir projects. In case you are looking for documentation for Elixir itself, [check out Elixir's website][elixir-lang].
 
+For documentation on writing documentation [check out Elixir's documentation][hex-writing-docs].
```

Comment:
I have to admit, it was kind of intentional 😈 , but I can see how it can be confusing